independent-projects/tools/devtools-common/pom.xml

```diff
@@ -57,6 +57,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
         </dependency>
+        <dependency>
```

Comment:
minor issue that can be fixed later. if this is just for parsing version devtools should already have maven GAV parser available - @aloubyansky ?

Reply:
We don't have a "home made" one. There is `org.apache.maven.artifact.versioning.DefaultArtifactVersion` that could have been used for this. I suppose @gsmet didn't want to depend on that API.

Reply:
It comes from `org.apache.maven:maven-artifact` which should already be on CP as a transitive dep.